daeun/lv1/두-정수-사이의-합.js

```diff
@@ -0,0 +1,21 @@
+function solution(a, b){
+    return (a+b)*(Math.abs(b-a)+1)/2; // 가우스 공식
+}
```

Comment:
```suggestion
function solution(a, b){
    return (a+b)*(Math.abs(b-a)+1)/2; // 가우스 공식
}
```
와우...가우스 공식을 떠올리지 못했습니다! 대운님 코드 잘봤습니다!

Reply:
@heejiyang 감사합니다! 상혁님과 유진님께도 말씀드렸듯이, 코드를 외우시면 다른 문제에서도 적용하실 수 있을 듯 합니다! 😃